Oleksii_Fedorchuk/HW14_Oleksii_Fedorchuk/task1.py

```diff
@@ -0,0 +1,94 @@
+class School:
+    def __init__(self, school_number: int, director: str, secretary: str, number_of_students: int):
+        self.__school_number = school_number
+        self.__director = director
+        self.__secretary = secretary
+        self.__number_of_students = number_of_students
+
+    def school_number(self):
+        return self.__school_number
+
+    @property
+    def director(self):
+        """ Encapsulation"""
+
+        return self.__director.capitalize()
+
+    @property
+    def secretary(self):
+        return self.__secretary
+
+    def number_of_students(self):
+        return self.__number_of_students
+
+    def school_bio(self):
+        return f"School number is: {self.__school_number}\n" \
+               f"Director of the schools is: {self.__director}\n" \
+               f"Secretary of the school is: {self.__secretary}\n" \
+               f"Total number of students in school is: {self.__number_of_students}\n"
+
+
+class Classes:
+    """ Polymorphism"""
+
+    def __init__(self, number_of_class: int, flor_in_the_school: int):
+        self.__number_of_class = number_of_class
+        self.__flor_in_the_school = flor_in_the_school
+
+    def number_of_class(self):
+        return self.__number_of_class
+
+    def flor_in_the_school(self):
+        return self.__flor_in_the_school
+
+
+class ClassOfMath(Classes):
```

Comment:
Why are all classes in 1 file? Could you please create each class in different module

Reply:
Done.